cli/src/commands/duplicate.rs

```diff
@@ -164,6 +170,26 @@ pub(crate) fn cmd_duplicate(
             }
         }
     }
+    let mut new_descs = HashMap::new();
+    if let Some(template) = tx
+        .settings()
+        .get_string("templates.duplicate_description")
+        .ok()
```

Comment:
nit: `.get_string(..)?` to propagate error

Reply:
This was a leftover from when I didn't put it in the config by default, thanks.